fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -511,8 +512,10 @@ Future<std::vector<std::pair<TenantName, TenantMapEntry>>> listTenantMetadataTra
 	wait(waitForAll(tenantEntryFutures));
 
 	std::vector<std::pair<TenantName, TenantMapEntry>> results;
-	for (auto const& f : tenantEntryFutures) {
-		results.emplace_back(f.get().tenantName, f.get());
+	state int idx = 0;
+	for (; idx < tenantEntryFutures.size(); ++idx) {
+		auto const& f = tenantEntryFutures[idx];
+		results.emplace_back(matchingTenants[idx].first, f.get());
```

Comment:
In data/standalone clusters, it shouldn't ever be the case that renameDestination is set, and so I think it should still work without changing this file